docs/src/main/sphinx/release/release-471.md

```diff
@@ -0,0 +1,66 @@
+# Release 471 (19 Feb 2025)
+
+## General
+
+* Add [](/functions/ai) for textual tasks on data using OpenAI, Anthropic, or
+  other LLMs using Ollama as backend.  ({issue}`25028`)
+* Include split count and total split distribution time in the `EXPLAIN ANALYZE`
+  output. ({issue}`25028`)
+* Add support for JSON logging format to console with `log.console-format=JSON`. ({issue}`25081`)
+* Add further packages for use with Python user-defined functions. ({issue}`25058`)
```

Comment:
What does "further packages" mean?

Reply:
There is a list of them in the docs .. a whole bunch of more Python libraries can now be included. I could list them if you want

Reply:
If it's a short list, yes. Otherwise, maybe rephrase to "Support additional python libraries ..."

Reply:
* `attrs`
* `bleach`
* `charset-normalizer`
* `defusedxml`
* `idna`
* `jmespath`
* `jsonschema`
* `pyasn1`
* `pyparsing`
* `python-dateutil`
* `rsa`
* `tomli`
* `ua-parser`

Reply:
I think that is too long

Reply:
Reworded